dataland-frontend/src/components/resources/frameworkDataSearch/ShowMultipleReportsBanner.vue

```diff
@@ -7,33 +7,30 @@
     </h4>
     <div id="reportList" style="display: flex">
       <span v-for="(report, name, index) in reports[indexOfNewestReportingPeriod]" :key="index" class="link-in-list">
-        <DocumentLink
-          data-test="documentLinkTest"
-          :download-name="name"
-          :fileReference="report.fileReference"
-          font-style="font-semibold"
-        />
+        <a @click="openReportDataTableModal(report, name as string)" class="link" :data-test="`report-link-${name}`">
+          <span>{{ name }}</span>
```

Comment:
In the backend change the name of the Report Date field to Publication Date also hence use the word Publication Date in the Frontend. This will require a trivial migration of data

Reply:
@ahoecherl the change in frontend is applied. for the backend renaming, if it is not urgent, we could take that as part of the story which involves the backend (the comment story). This way we do the migration just once